leapp/libraries/stdlib/__init__.py

```diff
@@ -132,7 +132,7 @@ def run(args, split=False, callback_raw=_logging_handler):
     result = None
     try:
         create_audit_entry('process-start', {'id': _id, 'parameters': args})
```

Comment:
I missed this the first time:

```suggestion
        create_audit_entry('process-start', {'id': _id, 'parameters': args, 'env': env})
```